### PR TITLE
Docs to get started with Docker, Mobile and Web Test Environment setups

### PR DESCRIPTION
**Pull request by mkasireddy**
Documentation and guidelines to get started with Dockerized Mobile and Web Test Environment setups

Jira: https://invisionapp.atlassian.net/browse/SILVER-816

### CODE REVIEW

**Comment by invisionjeremy** [comment]
@mkasireddy Can we add a section on how to add to the test suite?

File: getting_started_guide.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+[The mobile-web and Native App Test automation are implemented using Appium and the Instructions can be found here](http://appium.io/slate/en/master/?javascript#mobile-safari-on-a-real-ios-device)
+
+-  Web automation is not within Mobile testing scope, but nevertheless we needed to Implement the Web Test framework utilising [Webdriver.io](http://webdriver.io/api.html) for a bunch of Web features.
+    
+## Running Tests
+
+-  To kick off Web tests from within a Docker container, on a local docker container:
+
+```shell
+    ./node_modules/.bin/wdio wdio.conf.local.js
```
Comment (Jorshasaur) [inline getting_started_guide.md]: @mkasireddy is there a way we can run these commands from outside of the shell?  Maybe using npm?

File: mobile_test_setup.md [hunk 2]
```diff
@@ -31,7 +29,7 @@ With `mocha`, you can use any options you feel like. But you will need to specif
 `DEVICE`, which is a valid key into the `caps.js` file. For instance, you can run:
 
 ```shell
-DEVICE=android:4.4 mocha -t 180000 -R spec test/functional/Mobile/ios/web-specs.js
+DEVICE=ios:9.3 mocha -t 260000 -R spec test/functional/Mobile/ios/web-specs.js
```
Comment (Jorshasaur) [inline mobile_test_setup.md]: @mkasireddy is the `-t` command for the timeout?

File: getting_started_guide.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
+# Mobile Test Runner
+
+## Mobile Automated Test Environment setup
+
+[To get setup with the Docker container Environment](https://github.com/InVisionApp/mobile-test-runner/blob/master/README.md)
+
+[To get started with Automated Mobile tests](https://github.com/InVisionApp/mobile-test-runner/blob/master/getting_started_guide.md)
```
Comment (Jorshasaur) [inline getting_started_guide.md]: @mkasireddy I think you've got the doc linking back to itself here, probably not what you intended?

Comment (mkasireddy) [inline getting_started_guide.md]: ah good find @Jorshasaur thats not what I wanted to link.

File: mobile_test_setup.md (hunk 2 above)
Comment (mkasireddy) [inline mobile_test_setup.md]: @Jorshasaur yes. Im actually setting it as a global property. But when the set timeout isn't working Im overriding here

**Comment by Jorshasaur** [comment]
@mkasireddy here are my suggestions:

1. I think we need a clearer step by step guide on how to write a test, especially with the web portion. 
2. A best practices section would also be useful, with your experience what rules should we follow to prevent us from writing brittle tests?
3. What would the process be for handling errors, especially if this runs remotely?  Would we review the video on sauce labs and go from there?  I think we'd need to figure out a process.

File: getting_started_guide.md (hunk 1 above)
Comment (mkasireddy) [inline getting_started_guide.md]: @Jorshasaur Thats on my list of things to(simplify) look at

File: getting_started_guide.md (hunk 3 above)
Comment (datinc) [inline getting_started_guide.md]: Seems like the link is 404

**Comment by datinc** [comment]
Would be nice to have a readme or link to a webpage on how to create more tests. Did you use a framework to write the tests? If so could we have a link to have a look exposed API?

**Comment by mkasireddy** [comment]
@Jorshasaur @juansalas @datinc @invisionjeremy 

 I've Modified the getting_started.md and added the Screencast that walks you thru the Test creation and Test Run process. Please take a look!

File: getting_started_guide.md [hunk 4]
```diff
@@ -1,24 +1,31 @@
...
-[To get started with Automated Mobile tests](https://github.com/InVisionApp/mobile-test-runner/blob/master/getting_started_guide.md)
+## Mobile Automation Environment setup
 
-[The mobile-web and Native App Test automation are implemented using Appium and the Instructions can be found here](http://appium.io/slate/en/master/?javascript#mobile-safari-on-a-real-ios-device)
+-  New to Appium?? [get yourself aquainted with it here](http://appium.io/slate/en/master/?javascript#mobile-safari-on-a-real-ios-device)
 
--  Web automation is not within Mobile testing scope, but nevertheless we needed to Implement the Web Test framework utilising [Webdriver.io](http://webdriver.io/api.html) for a bunch of Web features.
+-  Appium API [here] (https://github.com/admc/wd/blob/master/doc/api.md)
+
+-  Get setup with Silver Team's Automated Mobile tests [Instructions are found here](https://github.com/InVisionApp/mobile-test-runner/mobile_test_setup.md)
```
Comment (invisionjeremy) [inline getting_started_guide.md]: I don't see a doc here @mkasireddy

Comment (Jorshasaur) [inline getting_started_guide.md]: @invisionjeremy she hasn't merged it into master yet, but I see it at https://github.com/InVisionApp/mobile-test-runner/blob/SILVER-816/mobile_test_setup.md

Comment (invisionjeremy) [inline getting_started_guide.md]: Ahh, I must have been looking at master. Thanks @Jorshasaur

File: getting_started_guide.md [hunk 5]
```diff
@@ -43,9 +50,16 @@
...
-                  .sleep(3000)
                   .elementByCss("div.legend a")
                   .click()
                   .text().should.include('Preview in Browser')
     };
-```
\ No newline at end of file
+```
+
+-  More detailed demo of creating an Automated Mobile-Web Test is explained in this [Screencast] (https://www.dropbox.com/s/502h2qm1pop4yk3/mobile_web_auto_demo.mov?dl=0)
```
Comment (invisionjeremy) [inline getting_started_guide.md]: @mkasireddy The screencast looks good. I cleaned up the audio and minimized the file here: Mobile Test Screencast-2.mp4

**Comment by invisionjeremy** [comment]
Looks good Manju 👍

**Comment by mkasireddy** [comment]
Thanks @invisionjeremy ! Merging away...

**Pull request merged**